cli/src/declarations.ts

```diff
@@ -490,6 +490,14 @@ export interface CapacitorConfig {
      */
     initialFocus?: boolean;
 
+    /**
+     * Whether to use hash-routing compatibility mode.
+     *
+     * @since 7.3.0
+     * @default false
+     */
+    hashRoutingCompatibility?: boolean;
```

Comment:
Hash routing (vanilla) works fine, the problem here seems to be with sveltekit router when using hash routing, so I don't think we should name it like that as other people might enable it thinking they need it when they don't. The name should probably be related to what it does, like `appendSlashToHostname` or something like that. And should probably be put inside the `server` object since all other url configuration options are in there.
The description should mention that it's for iOS only.